test/backward_compatibility/check_backward_compatibility.py

```diff
@@ -73,6 +73,7 @@
     ("aten::_foreach_div_", datetime.date(2021, 2, 25)),
     ("aten::_foreach_addcdiv", datetime.date(2021, 2, 25)),
     ("aten::mkldnn_linear", datetime.date(2021, 3, 2)),
+    ("aten::_mode", datetime.date(9999, 1, 1)),
```

Comment:
When choosing a date here, it should be just enough time to merge the changes. Not absurdly far in the future. I usually go a month or so ahead to prevent issues.

Reply:
Got it.